MagicalCryptoWallet.Tests/NodeBuilding/CoreNode.cs

```diff
@@ -60,7 +59,7 @@ public CoreNode(string folder, NodeBuilder builder)
 			_Builder = builder;
 			_folder = folder;
 			_state = CoreNodeState.Stopped;
-			CleanFolder();
+			CleanFolder().Wait();
```

Comment:
In case of `.Wait()` or `.Result` always use: `GetAwaiter().GetResult()` otherwise you get an `AggregateException`.

Reply:
That being said, of course using blocking (Wait, Result, GetAwaiter().GetResult()) code is already a compromise, since `await` should be used everywhere, but in this case this is only for testing so, oh well, it doesn't matter much.